README.md

```diff
@@ -1,21 +1,25 @@
-This is a small windows script to help extract Wings! amiga disks from the GOG release.
+This is a small Windows script to help extract Wings! amiga disks and the Kickstart 1.2 firmware from the GOG release. Also possible to extract from Linux, as explained below.
 
 # Requirements
 
-7z should be in your path, I got this on Windows by installing 7zip
+To have [7-Zip](https://www.7-zip.org/) installed.
 
 # Instructions
 
-1. Install Wings!
-2. find the Wings.exe, typically in "C:\GOG Games\Wings! Classic"
-3. drag and drop it over the extractwings.bat file
-4. you should now have two .adf files wings-1.adf and wings-2.adf these are the game disks
+1. Open the command prompt and run the command `set PATH=%PATH%;C:\Program Files\7-Zip\` (change `C:\Program Files\7-Zip\` to where you installed 7-Zip, if you didn't install it in the default folder)
+2. Download [extractwings's latest build](https://github.com/sigboe/extractwings/archive/refs/heads/master.zip)
+3. Unzip extractwings somewhere you can find
+4. Install Wings!
+5. Find the Wings.exe, typically in "C:\GOG Games\Wings! Classic"
+6. Drag and drop it over the extractwings.bat file
+7. You should now have three new files, wings-1.adf, wings-2.adf and Kick12.rom. The .adf files are your ROM, and Kick12.rom is the firmware file.
 
 # Linux
-For Linux, things are simpler
+For Linux, things are simpler. Just run these commands from the folder with Wings.exe using the terminal:
```

Comment:
I prefer not suggesting the directory to run the commands from.

Reply:
I see. I'll remove it, then. Hopefully people know the commands need to be run from where the Wings.exe file is, or to point the command to the right folder.
Also, I just remembered these commands could possibly be compressed into a .sh script on Linux. I'll be testing it after doing the changes in the PR.

Reply:
About the .sh script, it's surprisingly easy.
Just create a .sh file (or a .txt file and rename the extension .sh) and paste the commands in the README.md, then run this command with super user privileges ("sudo" argument in Ubuntu, not sure on other distros):
```chmod +x /path/to/script.sh```
After that, put the .sh file in the same folder as Wings.exe and either open the file directly if your OS allows, or run in the terminal (`cd /path/to/folder/with/the/files/` then `./script.sh`). The files should be extracted in the same folder.

Reply:
Maybe you could add such .sh script to the downloadable files as well? (I'm not sure if/how I can add files through PR, just edit existing ones)